src/core/string_or_view.cc

```diff
@@ -0,0 +1,47 @@
+// Copyright 2024, DragonflyDB authors.  All rights reserved.
+// See LICENSE for licensing terms.
+//
+
+#include "core/string_or_view.h"
+
+namespace dfly {
+using namespace std;
+
+StringOrView StringOrView::FromString(string s) {
+  StringOrView sov;
+  sov.val_ = std::move(s);
+  return sov;
+}
+
+StringOrView StringOrView::FromView(string_view sv) {
+  StringOrView sov;
+  sov.val_ = sv;
+  return sov;
+}
+
+bool StringOrView::operator==(const StringOrView& o) const {
```

Comment:
nit: i would move onelines to class definition and others to the header file and get rid of cc.